### ISSUE TITLE
Rules are disabled after a emqx two nodes cluster restart

### ISSUE THREAD

**Issue by ctellechea2001** [emqx/emqx#9618]
### What happened?

We have installed emqx 4.3.8 in kubernetes. Currently we have 4 rules that those are enabled. We have a two nodes emqx cluster

**sbx-emqx-0                                               1/1     Running            0          101m
sbx-emqx-1                                               1/1     Running            0          15h**

We discovered that when both pods are restarted at the same time (we have automatically shutdown/startup for lower enviroment like sbx/dev) the rules started with **status disabled**. Please see the images.

![image](https://user-images.githubusercontent.com/45426180/209671311-d7e109bf-4418-4736-b68f-a7eeaba33417.png)

Do you have any workaround to fix that?

### What did you expect to happen?

Keep the rules enabled after a emqx cluster restart (Just to add, we have persistence in cluster with ebs volumes (pvc).

### How can we reproduce it (as minimally and precisely as possible)?

Cluster in kubernetes with 2 nodes, with **podManagementPolicy: Parallel** that you suggested in kubernetes. Shutdown the nodes and start again.

### EMQX version

<details>

```console
$ ./bin/emqx_ctl broker
bash-5.0$ ./bin/emqx_ctl broker
sysdescr  : EMQ X Broker
version   : 4.3.8
uptime    : 1 hours, 47 minutes, 23 seconds
datetime  : 2022-12-27 13:18:22
bash-5.0$
```

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Log files

<details>

</details>

**Comment by tigercl** [comment]
Hi, @ctellechea2001 
This will happen if the external resources are not ready in time when EMQX starts, you can upgrade to the latest version, we have fixed this issue.

**Comment by ctellechea2001** [comment]
Thanks @tigercl for your support, It is possible to jump from 4.3.8 to latest without issues in out kubernetes chart that is working fine?

**Comment by tigercl** [comment]
5.0 has relatively large changes. This bug has been fixed in the latest versions of both 4.3 and 4.4.

**Comment by ctellechea2001** [comment]
@tigercl thanks a lot!! We will try to upgrade to 5.0
